src/docs/index.adoc

```diff
@@ -15,12 +15,12 @@
 ** link:PSA/PSAUserGuide.html[*Service Automation*] (PaaS On-Demand, Service deployment and management)
 * link:admin/ProActiveAdminGuide.html[*PWS Admin Guide*] (Installation, Infrastructure & Nodes setup, Agents,…​)
 
-=== Machine Learning Open Studio (MLOS)
+=== Machine Learning Open Studio (PML)
 
-* link:MLOS/MLOSUserGuide.html[*MLOS User Guide*] (​a complete Data Science and Machine Learning Platform)
+* link:PML/PMLUserGuide.html[*PML User Guide*] (​a complete Data Science and Machine Learning Platform)
```

Comment:
​a complete Data Science and Machine Learning Platform -> a complete Data Science and Machine Learning platform, with Studio & MLOps.